src/set_theory/cardinal.lean

```diff
@@ -881,6 +881,12 @@ succ_le.1 $ by rw [← nat_succ, ← lift_mk_fin, omega, lift_mk_le.{0 0 u}]; ex
 @[simp] theorem one_lt_omega : 1 < ω :=
 by simpa using nat_lt_omega 1
 
+@[simp] theorem two_lt_omega : 2 < ω :=
+by simpa using nat_lt_omega 2
+
+@[simp] theorem three_lt_omega : 3 < ω :=
+by simpa using nat_lt_omega 3
```

Comment:
Oddly I cannot just use `simpa using nat_lt_omega 3` where I now have to use `exact three_lt_omega`. If you don’t think we should add this lemma here, I’d appreciate help proving it without it :-)

Reply:
Ah, the linter suggests `simp only [cardinal.bit1_lt_omega, cardinal.one_lt_omega]`, I'll try that when at the laptop again.

Reply:
Trying that I get
```
invalid simplification lemma 'cardinal.bit1_lt_omega' (use command 'set_option trace.simp_lemmas true' for more details)
```
:shrug: